### ISSUE TITLE
When a vehicle is selected, only the drivers for that vehicle must appear.

### ISSUE THREAD

**Issue by niharRaut** [gaurav-naik/andesit_karang_anyar#4]
"Add filter query to Driver field in WBW doctype. 
(Post WB-Link: WBW will be auto-created at PO / SO. Driver will be set automatically from WB data.)"  
**ETA:** 0.5 hours

**Comment by gaurav-naik** [comment]
@saidsl
Hi Said.

A child table has been added under the Link Drivers section for linking drivers to a vehicle in the Vehicle doctype.
In Weighbridge Weighing, drivers are being filtered by Vehicle.

Let me know if you have any suggestions.

**Comment by saidsl** [comment]
Hi Gaura,

I just checked what has been done and that perfectly fine. Works as it should. Nice & resolved solution

Thanks
Said


On 19 Jul 2016, at 1:38 PM, gaurav-naik <notifications@github.com> wrote:

@saidsl <https://github.com/saidsl>
Hi Said.

A child table has been added under the Link Drivers section for linking drivers to a vehicle in the Vehicle doctype.
In Weighbridge Weighing, drivers are being filtered by Vehicle.

Let me know if you have any suggestions.
